apps/dotcom/src/components/DocumentName/DocumentName.tsx

```diff
@@ -148,14 +150,19 @@ function DocumentTopZoneContainer({ children }: { children: ReactNode }) {
 		const xCoordIfLeftAligned = leftWidth + MARGIN_BETWEEN_ZONES
 
 		const left = element.offsetLeft
-		const xCoord = Math.max(xCoordIfCentered, xCoordIfLeftAligned) - left
 		const maxWidth = Math.min(
 			totalWidth - rightWidth - leftWidth - 2 * MARGIN_BETWEEN_ZONES,
 			MAX_TITLE_WIDTH_PX
 		)
+		const xCoord = Math.max(xCoordIfCentered, xCoordIfLeftAligned) - left
 
+		// Squeeze the title if the right panel is too wide on small screen
+		if (rightPanel.offsetWidth > STYLE_PANEL_WIDTH && totalWidth < 841) {
```

Comment:
can you make these numbers into constants? 841 is very specific and we won't know what it means later on :)
same for the 60 below

Reply:
Done, I had to go and figure out why it was that 60 worked and ended up changing it to 52.